### PR TITLE
Including database scripts into rpm

### PR DESCRIPTION
**Pull request by jmmovilla**
@jesuspg @flopezag

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -473,6 +473,16 @@
                                                 ${project.build.directory}/paas-manager-server-dist/webapps/paasmanager.xml
                                             </location>
                                         </source>
+                                        <source>
+                                            <location>
+                                                ${project.build.directory}/paas-manager-server-dist/resources/db-initial.sql
```
Comment (flopezag) [inline pom.xml]: Sorry but whera are those files (db-initial.sql and db-changelog.sql)?

**Comment by jesuspg** [comment]
LGTM

File: pom.xml (hunk 1 above)
Comment (jmmovilla) [inline pom.xml]: Initially in migrations/src/main/resources . They should be passed to /opt/fiware-paas/resources to créate database schema

Comment (flopezag) [inline pom.xml]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**